client/extensions/woocommerce/app/promotions/promotion-models.js

```diff
@@ -19,26 +19,28 @@ import PromotionAppliesToField from './fields/promotion-applies-to-field';
 /**
  * "Coupon code" field reused for all coupon promotion types.
  */
-const couponCodeField = {
-	component: TextField,
-	labelText: translate( 'Coupon code' ),
-	explanationText: translate(
-		'Only apply this promotion when the customer supplies the coupon code'
-	),
-	placeholderText: translate( 'Enter coupon code' ),
-	isRequired: true,
-};
+const couponCodeField = (
+	<TextField
+		labelText={ translate( 'Coupon code' ) }
+		explanationText={ translate(
+			'Only apply this promotion when the customer supplies the coupon code'
+		) }
+		placeholderText={ translate( 'Enter coupon code' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **51** times:
`translate( 'Enter Coupon Code' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).